terraform/destroy.sh

```diff
@@ -28,6 +28,16 @@ acct=$(gcloud info --format="value(config.account)")
 SCRIPT_DIR=$(realpath $(dirname "$0"))
 cd $SCRIPT_DIR
 
+# For the purposes of telemetry
+# session is defined as the current time the user runs destroy.sh file
+SESSION=$(python3 -c "import uuid; print(uuid.uuid4())")
```

Comment:
Are you sure you always want the session to be unique for each script run? You could try to save the session id as an environment variable if you want it to span multiple scripts

Reply:
Yes, actually that sounds like a much better approach to what I want to do! I will try to do that.